code/src/components/Styles.js

```diff
@@ -0,0 +1,92 @@
+import styled from 'styled-components/macro';
+
+export const Container = styled.div`

```

Comment:
You seem to have a very good understanding on how to style with styled components which is a treat for the eye. 
You have good names to them which makes it easy to follow where the html-tags are replaced with your styling components. Be proud!